src/locales/en.locale.json

```diff
@@ -519,6 +521,9 @@
   "mno_enterprise.templates.dashboard.marketplace.show.comment.delete_modal_body": "Do you really want to delete your comment?",
   "mno_enterprise.templates.dashboard.marketplace.show.comment.delete_modal_cancel": "Cancel",
   "mno_enterprise.templates.dashboard.marketplace.show.comment.delete_modal_delete": "Delete",
+  "mno_enterprise.templates.dashboard.marketplace.show.comment.hide": "Hide Comments",
+  "mno_enterprise.templates.dashboard.marketplace.show.comment": "Comment",
+  "mno_enterprise.templates.dashboard.marketplace.show.comments": "Comments",
```

Comment:
You can use pluralisation to manage this (https://angular-translate.github.io/docs/#/guide/14_pluralization)